content/account/organizations.textile

```diff
@@ -0,0 +1,178 @@
+
+<aside data-type='important'>
+	<p>Ably will reject any SCIM-provisioned user whose email address hasn't been pre-approved and added to your organization by Ably support. Ensure all users you intend to provision are registered with Ably beforehand.</p>
+	</aside>
+
+h3(#app). Setup an identity provider's app
+
+To "group permissions":#permissions and "provision":#provision users with SCIM you need to enable "SSO":/account/sso with your chosen identity provider.
+
+h3(#group). Group permissions via SCIM
+
+When you enable "SSO":/account/sso with your chosen identity provider, you assign users and groups to the new identity provider's app. The group you created in identity provider's app is available to view and for user access management within Ably:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Users*.
+** Under *Ably Realtime identity provider groups*, click *Manage account access*.
+** Select the user's account access you want to manage.
+** Specify the user's required *Permissions* for the organization.
+** Assign the necessary *Roles* for required *Account* access.
+
+
+<aside data-type='note'>
+<p>When you adjust a user's individual roles and permissions, permissions assigned via groups appear greyed out, indicating they cannot be modified individually. </p>
+</aside>
+
+h3(#provision). Provision users
+
+Provisioning users enables you to grant multiple users access to your organization, either automatically via "SCIM":#scim or "manually":#manual through user invitations.
+
+<aside data-type='note'>
+<p>Provisioning users via SSO is recommended over manual invitation because it strengthens security by centralizing access control, enhances operational efficiency through automation, and improves the user experience with quicker access. Provisioning users via SSO also ensures compliance with regulatory requirements, and reduces the risk of human error.</p>
+</aside>
+
+h4(#ably). Configure Ably SSO
+
+"Configuring SSO":/account/sso#configure between Ably and your chosen identity provider enables authentication by linking Ably's SAML-based SSO with the identity provider. This setup allows users to access Ably through their organizational credentials.
+
+h4(#copy). Copy Ably SSO configuration values
+
+Start the configuration process by copying Ably's SSO configuration values:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Settings*.
+** Under *Single Sign-On (SSO)* enable SSO.
+** Copy the *Single sign-on URL* and *Audience URI (SP Entity ID):* values.
+
+h4(#complete). Complete Ably SSO Configuration
+
+Finish the configuration process via Ably:
+
+* In your Ably account dashboard, go back to *Settings* > *Authentication Settings*.
+* Paste and *Save* the values obtained from your identity provider's app:
+** *Identity Provider Single Sign-On URL*.
+** *Identity Provider Issuer*.
+** *X.509 Certificate* and *Save SSO Settings*.
```

Comment:
We're linking out to SSO instructions on a different page, and then repeating them again here which seems a bit odd. Do we need to have them repeated here?